eng/pipelines/templates/steps/verify-autorest.yml

```diff
@@ -0,0 +1,66 @@
+parameters:
+  - name: ServiceToTest
+    type: string
+    default: ''
+  - name: SDKType
+    type: string
+    default: 'client'
+  - name: ServiceDirectory
+    type: string
+  - name: NodeVersion
+    type: string
+    default: '12.x'
+  - name: PythonVersion
+    type: string
+    default: '3.9'
+  - name: auto_rest_clone_url
+    type: string
+    default: 'https://github.com/Azure/autorest.python.git'
+
+steps:
+  - task: UsePythonVersion@0
+    displayName: Use
+    inputs:
+      versionSpec: ${{ parameters.PythonVersion }}
+
+  - task: NodeTool@0
+    inputs:
+      versionSpec: ${{ parameters.NodeVersion }}
+
+  - script: |
+      npm install -g autorest
+      autorest --help
+    displayName: "Install autorest"
+
+  - script: |
+      python --version
+      python -m pip install pip==20.1
```

Comment:
We have pip pinned in `ci_tools.txt`. It's 20.2 there. I understand why you have it separate here, as you don't need every single package that is in `ci_tools.txt`.